packages/wdio-cli/src/config.js

```diff
@@ -176,34 +176,61 @@ export const QUESTIONNAIRE = [{
     message: 'What is the port on which that service is running?',
     default: '80',
     when: (answers) => answers.backend.indexOf('different service') > -1
+}, {
+    type: 'input',
+    name: 'env_user',
+    message: 'Environment variable for username',
+    default: 'BROWSERSTACK_USER',
+    when: (answers) => answers.backend.startsWith('In the cloud using Browserstack')
+}, {
+    type: 'input',
+    name: 'env_key',
+    message: 'Environment variable for access key',
+    default: 'BROWSERSTACK_ACCESSKEY',
+    when: (answers) => answers.backend.startsWith('In the cloud using Browserstack')
 }, {
     type: 'input',
     name: 'env_user',
     message: 'Environment variable for username',
     default: 'SAUCE_USERNAME',
-    when: (answers) => answers.backend.indexOf('In the cloud') > -1
+    when: (answers) => answers.backend === 'In the cloud using Sauce Labs'
 }, {
     type: 'input',
     name: 'env_key',
     message: 'Environment variable for access key',
     default: 'SAUCE_ACCESS_KEY',
-    when: (answers) => answers.backend.indexOf('In the cloud') > -1
+    when: (answers) => answers.backend === 'In the cloud using Sauce Labs'
```

Comment:
Do you find it reasonable to move repetitive messages like `'In the cloud using Sauce Labs'`  to constants?

Reply:
I did that so I can add some custom questions for Sauce Labs specific questions.